### PR TITLE
Adress formatting review

### PR DESCRIPTION
**Pull request by XavierJp**

### CODE REVIEW

File: clients/rna/index.ts [hunk 1]
```diff
@@ -74,15 +74,14 @@ export const mapToDomainObject = (
...
-    adresse: formatAdresse(
-      '',
-      adresse_numero_voie,
-      adresse_repetition,
-      adresse_type_voie,
-      adresse_libelle_voie,
-      adresse_code_postal,
-      adresse_libelle_commune
-    ),
+    adresse: formatAdresse({
```
Comment (HAEKADI) [inline clients/rna/index.ts:77]: So the RNA API returns only these fields ?

File: clients/sirene-insee/siret.ts [hunk 2]
```diff
@@ -221,16 +238,19 @@ export const mapEtablissementToDomainObject = (
...
+      numeroVoie: numeroVoieEtablissement,
+      indiceRepetition: indiceRepetitionEtablissement,
+      typeVoie: typeVoieEtablissement,
+      libelleVoie: libelleVoieEtablissement,
+      codePostal: codePostalEtablissement,
+      libelleCommune: libelleCommuneEtablissement,
+      distributionSpeciale: distributionSpecialeEtablissement,
+      codeCedex: codeCedexEtablissement,
+      libelleCommuneCedex: libelleCedexEtablissement,
+      pays: libellePaysEtrangerEtablissement,
```
Comment (HAEKADI) [inline clients/sirene-insee/siret.ts:252]: I did not use `pays` or the other country fields. Should I add them ?

Comment (HAEKADI) [reply clients/sirene-insee/siret.ts:252]: @XavierJp

Comment (XavierJp) [reply clients/sirene-insee/siret.ts:252]: yes !

**Pull request merged**